aspnet/mvc/views/layout.rst

```diff
@@ -0,0 +1,98 @@
+Layout
+=======
+By `Steve Smith`_
+
+Views can take advantage of Layout, ViewStart, and ViewImports files to share common elements.
```

Comment:
Maybe a bit too specific to the file. Expand this to be more descriptive of what this article enables a user to accomplish (without mentioning the files specifically, layouts kind of mess the whole specific naming thing up).